morphocut/str.py

```diff
@@ -6,6 +6,9 @@
 
 @Output("string")
 class Format(Node):
+    """Create the string according to our given format
+    For example, if we want to append a string in a certain order or want to replace our arguments
+    """
```

Comment:
https://trello.com/c/ub9i0YrA/14-write-a-docstring

Do not write "for example...", write a proper code example.

Reply:
In which order are the arguments given to str.format? What happens if I supply the same key twice, once in _kwargs, once in kwargs?